lib/build.js

```diff
@@ -214,6 +214,14 @@ function createModel(pathResolver, modelConfig, options, config, jsBundleOptions
                 cssBundleOptions
             )
         )
+        .then(() => !options.allowDownload && utils.section('Disabling report download option in bundles', () => {
+            fs.writeFileSync(
+                pathResolver('model.js'),
+                fs.readFileSync(pathResolver('model.js'), 'utf8')
+                    .replace(new RegExp(`function ${utils.MODE_PLACEHOLDER}\\(\\){return.*?}\\(\\)`), '"single"')
+                    .replace(new RegExp(`function ${utils.SETUP_MODEL_PLACEHOLDER}\\(\\){return.*?}\\(\\)`), JSON.stringify({ ...config.models[0], download: false, cache: false}))
```

Comment:
I think will be better to patch config instead.

Reply:
You mean `setup.js`? I considered such a solution when I was developing a static report download. But when we generate setup.js we're not sure about where this file goes. This will overcomlicate `gen[]` system.

Reply:
Patch config of `makeBundles`, yeah this is a simpler way.